src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +23,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(string -> string.split(SEPARATOR))
+                .flatMap(Arrays::stream)
```

Comment:
these two calls could be merged the next way:
```suggestion
                .flatMap(numbers -> Arrays.stream(numbers.split(SEPARATOR)))
```